📝 add docusaurus documentation (#266)
ajnart/homarr: docs/docs/customizations/custom-search-engine.md

---
tags:
  - Customization
  - Design
  - Background
  - Custom
---

# Customize your Search Engine
## Select a Search Engine

> If you use the search box module, you might want to configure your search engine.

Reviewer: ```suggestion
If you use the Search Module, you might want to configure a custom search engine.
```

> Open your settings on the top right.

Reviewer: ```suggestion
Open settings by clicking the settings cog on the top right.
```

> You can select from a default list of search engine, but you can also use your own.

Reviewer: ```suggestion
You can either select one of the default search engines, or you can provide a custom one.
```
